dist/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do list</title>
+<script defer src="main.bundle.js"></script></head>
+
+<body>
+    <main>
+        <section class="container">
+            <div class="ctn-head">
+                <h5 class="head-title">Today's To Do</h5>
+                <div class="ctn-icon"><i class="fa-solid fa-arrows-rotate"></i></div>
+            </div>
+            <div class="line"></div>
+            <div class="ctn-input">
+                <input id="input-txt" class="input-txt" type="text" placeholder="Add to your list...">
+                <div id="ctn-icon-arrow" class="ctn-icon"><i class="fa-solid fa-arrow-turn-down"></i></div>
+            </div>
+            <div class="line"></div>
+            <ul id="ctn-task-list" class="ctn-task-list">
+
+                <!-- I left comments for test purpose. I will delete it for the last submit -->
+
+                <!-- <li>
+                    <div class="ctn-task">
+                        <div class="ctn-left">
+                            <input class="checkbox" type="checkbox" name="completed">
+                            <input id="description" class="description" type="text" placeholder="Something interesting" value="cwvev">
+                        </div>
+                        <div class="ctn-icon">
+                            <div class="ctn-opt">
+                                <i class="fa-solid fa-ellipsis-vertical"></i>
+                            </div>
+                            <div class="del-opt">
+                                <i class="fa-solid fa-trash-can"></i>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="line"></div>
+                </li> -->
```

Comment:
**Good job** on making the file readable and simple, but next time try to **remove** comments because they look **unprofessional** and don't follow the **best practices**. 👍